QUARKUS-732.md

```diff
@@ -0,0 +1,33 @@
+# QUARKUS-732 - Reduce configuration when deploying to OpenShift
+
+The goal of this test plan is to cover usage of Service Bindings[5] for managed Kafka on OpenShift.
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-732
+
+### Scope of the testing
+ - Verify, that following properties can be replaced with Service Bindings
+   - `%prod.kafka.bootstrap.servers`
+   - `%prod.kafka.security.protocol`
+   - `%prod.kafka.sasl.mechanism`
+   - `%prod.kafka.sasl.jaas.config`
+   - `%prod.quarkus.openshift.env.secrets`
+   - `quarkus.openshift.env.mapping.kafka-username.from-secret`
+   - `quarkus.openshift.env.mapping.kafka-username.with-key`
+   - `quarkus.openshift.env.mapping.kafka-password.from-secret`
+   - `quarkus.openshift.env.mapping.kafka-password.with-key`
+
+### Impact on test suites and testing automation
+- Additional module in the test suite for OpenShift testing
+- These tests require deployment of Managed Kafka on OpenShift. This is not possible at this time, but we expect it to be implemented later.
```

Comment:
This is also documented here: 

https://github.com/quarkus-qe/quarkus-test-plans/blob/main/QUARKUS-1027.md#future-considerations

We should somehow reference this in this doc also, just to have a nice history of our plans.